tests/t_server_null_default.rc

```diff
@@ -0,0 +1,65 @@
+# Notes regarding --dev null server and client configurations:
```

Comment:
```suggestion
# -*- shell-script -*-
# Notes regarding --dev null server and client configurations:
```

Reply:
We seem to have similar declarations/hints elsewhere, namely `-*- Autoconf -*-` in .m4 files and `-*- coding: utf-8 -*-` in Python files. I see how this *might* be useful in some context as this is a shell script without an explicit `#!/bin/sh` at the top. However, I think we should also add this to *t_client.rc-sample* to be consistent with these markers. Thoughts?

Reply:
Can't hurt